apps/judicial-system/web/src/components/CaseInfo/CaseInfo.tsx

```diff
@@ -86,6 +86,16 @@ const Defendants: React.FC<React.PropsWithChildren<Props>> = ({
   )
 }
 
+export const Prosecutor: React.FC<Props> = ({ workingCase }) => {
+  const { formatMessage } = useIntl()
+  return (
+    <Entry
+      label={formatMessage(core.prosecutorPerson)}
+      value={workingCase.prosecutorsOffice?.name || ''}
```

Comment:
Consider using `??` as you only want to set to the empty string when null or undefined.